doc/Programming-guide/Key-platform-features/Events/index.md

```diff
@@ -4,4 +4,206 @@ title: Events
 permalink: /:path/
 sort_idx: 80
 ---
-Purpose of events. Event classes and class families. Event TTL.
+
+* [Event generation and handling overview](#event-generation-and-handling-overview)
+* [Event class and event class schema](#event-class-and-event-class-schema)
+* [Event class families](#event-class-families)
+* [Event family mapping](#event-family-mapping)
+* [Event routing](#event-routing)
+* [Event sequence number](#event-sequence-number)
+* [SDK generation](#sdk-generation)
+
+
+
+The Kaa Event subsystem enables generation of events on endpoints in near real-time fashion, handling those events on a Kaa server, and dispatching them to other endpoints that belong to the same user (potentially, across different applications). The Kaa event structure is determined by a configurable event class schema.
+
+The Kaa Event subsystem provides the following features.
+
+* Generation of the event object model and related API calls in the endpoint SDK
+* Enforcement of data integrity and validity
+* Efficient targeting of event recipients
+* Efficient and compact serialization
+
+It is the developer's responsibility to design event class schemas and make the client application interpret event data supplied by the endpoint library. The Kaa administrator, in turn, can provision those schemas into the Kaa server and generate the endpoint SDK.
+
+
+## Event generation and handling overview
+
+![](images/EventGeneration2.png)
```

Comment:
Please add some details. For example, "process of event generation and handling described on following diagram". Move "It is the developer's responsibility to design event class schemas and make the client application interpret event data supplied by the endpoint library. The Kaa administrator, in turn, can provision those schemas into the Kaa server and generate the endpoint SDK." to this section below the diagram. Add smth about scope of event exchange (user ids) and link to Owner verifiers page